basic-example-dockerfile/{{cookiecutter.project_name}}/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim-buster
+FROM python:3.10-slim-buster
```

Comment:
The wine classification template uses this base image, so I assumed it was safe to use here too, but we can switch back if need be.